tsconfig.json

```diff
@@ -3,8 +3,6 @@
     "target": "ESNext",
     "module": "ESNext",
     "moduleResolution": "Bundler",
-    // Allow importing TypeScript files using their native extension (.ts(x)).
-    "allowImportingTsExtensions": true,
```

Comment:
In latest typescript this only works with emitDeclarationOnly, but imo this flag is kinda tech debt anyways because it relies on bundler tool to resolve the import paths, and we actually want to stay bundleless.